<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doc_form</class>
 <widget class="QWidget" name="doc_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>781</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1083</width>
    <height>781</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1083</width>
    <height>781</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="doc_name">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Заказ на перемещение 000000001 от 09.10.2020</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_storage">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Склад:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="storage">
           <property name="text">
            <string>Склад1</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_status">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Статус:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status">
           <property name="text">
            <string>На исполнение</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_execute_to">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Выполнить до:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="execute_to">
           <property name="text">
            <string>08.10.2020 17:30</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_sending_date">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Задание получено:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="date_sending">
           <property name="text">
            <string>08.10.2020 13:45:02</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_team_leader">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Кладовщик:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="team_leader">
           <property name="text">
            <string>Пупкин И.И.</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_start_time">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Начало выполнения:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="start_time">
           <property name="text">
            <string>13:08</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_type">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Вид задания:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="type">
           <property name="text">
            <string>Сборка</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1" rowspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_team_number">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Бригада №:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="team_number">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="2" rowspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_end_time">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Конец выполнения:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="end_time">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_destination">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Направление:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="destination">
           <property name="text">
            <string>Склад Армавир</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_autos_number">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Номер авто:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="autos_number">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>х777хх123</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="doc_tbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Номенклатура</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Код</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>Количество</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>Статус</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>Фанера ФК 10 * 1525 * 1525 мм сорт 2/2 шл 2 ст береза Муром (1 шт = 2,325625 м2)</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>УТ-00015793</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>Фанера ФК 10 * 1525 * 1525 мм сорт 2/2 шл 2 ст береза Муром (1 шт = 2,325625 м2)</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>УТ-00015793</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>Исполнено</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
